guidelines/sc/22/focus-appearance.html

```diff
@@ -23,15 +23,16 @@ <h4>Focus Appearance</h4>
         </li>
     </ol>
 
-    <p>Where a user interface component has active sub-components, if a sub-component receives a focus indicator, these requirements may be applied to the sub-component instead.</p>
-
     <p>Exceptions:</p>
     <ul>
         <li>The focus indicator is determined by the user agent and cannot be adjusted by the author, or</li>
         <li>The focus indicator and the indicator's background color are not modified by the author.</li>
     </ul>
-                
-    <p class="note">Note: Examples of sub-components that may receive a focus indicator are menu items in an opened drop-down menu. However, it may also be possible to indicate user interaction for such sub-components by relying strictly on a visual indication of which item is <em>selected</em>. Where selectable sub-components have no differentiated focus indicator, the visual indicator for sub-component selection is measured against <a href="https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html">1.4.11 Non-text Contrast</a> requirements, not against this Criterion.</p>
-    <p class="note">Note: Contrast calculations can be based on colors defined within the technology (such as HTML, CSS and SVG). Pixels modified by user agent resolution enhancements and anti-aliasing can be ignored.</p>
+    
+    <p class="note">What is perceived as the user interface component and any sub-components (to determine enclosure or size) depends on its visual presentation. The visual presentation includes, but is not limited to the component's visible content, border, component-specific background, and effects emanating from the visible component such as shadows or glow effects.</p>
```

Comment:
I am suggesting this modification because selectable sub-components are focused. They may be focused and selected, but focused doesn't stay on the top-level component. See https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#x6-3-focus-vs-selection-and-the-perception-of-dual-focus

Reply:
That's a useful resource, but as it is says "in some widgets, moving focus may also perform the select operation."

I.e. from a user point of view they are the same. 

Yet, when moving away the selected things stays selected. So if we require a focus and select indicators at the same time (and the selection has to meet non-text contrast), we setup a 3-color contrast problem.